cmd/controller/app/options/options.go

```diff
@@ -199,6 +204,14 @@ var (
 		csrvenaficontroller.CSRControllerName,
 		csrvaultcontroller.CSRControllerName,
 	}
+	// Annotations that will be copied from Certificate to CertificateRequest and to Order.
+	// By default, copy all annotations except for the ones applied by kubectl, fluxcd, argocd.
+	defaultCopiedAnnotations = []string{
+		"*",
+		"-kubectl.kubernetes.io/",
+		"-fluxcd.io/",
+		"-argocd.argoproj.io/",
+	}
```

Comment:
These look like prefixes, rather than the whole annotation itself. Can we update the variable names and flag to make it clear these are _prefixes_. (i.e. `CopiedAnnotationPrefixes`).

Reply:
Thanks, that makes sense